media-video/blinkenthemes/blinkenthemes-0.10-r1.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Themes for blinkensim"
+HOMEPAGE="http://blinkenlights.net/project/developer-tools"
+SRC_URI="http://blinkenlights.de/dist/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="media-libs/blib"
```

Comment:
not needed in `RDEPEND`? please check linking.

Reply:
checked.. this one is clearly not needed for ```RDEPEND```. It only installs mostly ```png``` files with some other files..